pkg/storage/cache/cache.go

```diff
@@ -76,6 +76,24 @@ func (cache *Cache) Flush() {
 	<-cache.cleanupDone
 }
 
+func (cache *Cache) Delete(key string) {
+	cache.Flush()
```

Comment:
This causes an `EvictionChannel` to be closed. How to remove the entries from cache in different way?

Reply:
`Flush` will clear the whole cache if I understand it correctly. I don't see a `Delete` function in [lfu-go](https://github.com/dgrijalva/lfu-go/blob/master/lfu.go) which is a bummer. It should be easy to add though.

We can add it to our fork of lfu-go https://github.com/pyroscope-io/lfu-go The function should just do something like this:
```
c.lock.Lock()
defer c.lock.Unlock()
delete(c.values, key)
```

To make the project use our fork we'll need to make changes to go.mod the same way we use this revive fork:
https://github.com/pyroscope-io/pyroscope/blob/main/go.mod#L48

Reply:
allright, I have it done locally, but I need write access to lfu-go to commit it. I have updated cache.go, but I expect test to fail until the fork will be updated.